docs/scipp-neutron/groupby.ipynb

```diff
@@ -12,7 +12,11 @@
     "## Grouping with bins\n",
     "\n",
     "Note that this notebooks requires [Mantid](https://www.mantidproject.org/Main_Page) and data files.\n",
-    "A [binder](https://mybinder.org/v2/gh/scipp/scipp-neutron-jupyter-demo/master) is available that can run this notebook."
+    "A [binder](https://mybinder.org/v2/gh/scipp/scipp-neutron-jupyter-demo/master) is available that can run this notebook.\n",
+    "\n",
+    "[PG3_4844_event.nxs](http://198.74.56.37/ftp/external-data/MD5/d5ae38871d0a09a28ae01f85d969de1e) is required for this notebook.\n",
```

Comment:
Maybe better to refer to `data/fetch_neutron_data.py`

Reply:
Yes I don't know what is better... I think there are some hard-coded paths in the script, so not sure it's good to point to it now. Maybe if we make it more generic/flexible, we could change the docs then?

Reply:
I'm going to leave this as is